build/commands/lib/config.js

```diff
@@ -88,7 +88,7 @@ const Config = function () {
   this.defaultGClientFile = path.join(this.rootDir, '.gclient')
   this.gClientFile = process.env.BRAVE_GCLIENT_FILE || this.defaultGClientFile
   this.gClientVerbose = getNPMConfig(['gclient_verbose']) || false
-  this.targetArch = getNPMConfig(['target_arch']) || 'x64'
+  this.targetArch = getNPMConfig(['target_arch']) || process.arch || 'x64'
```

Comment:
this doesn't make sense to me, it would never fall back to `x64`. I think we should probably just drop `|| 'x64'` here

Reply:
done